### PR TITLE
Borgs Can Adjust Suit Sensors

### LINKED ISSUE #25536: Borgs can't adjust suit sensors
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1619

### Issue Description

I remember borgs used to be able to adjust suit sensors. I drag their body to myself, and next to their jumpsuit is a little shirt icon that says "Adjust suit sensors". Pressing it does nothing. By comparison, the option "Remove accessory" DOES work.

### What did you expect to happen?

_No response_

### What happened instead?

_No response_

### Why is this bad/What are the consequences?

_No response_

### Steps to reproduce the issue.

Already explained

### When did the problem start happening?

When I started playing again a few days ago, after a long hiatus

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Fixes #25536.

Also cleans up the proc a little. Adds some documentation. ::)

## Why It's Good For The Game
Oversight bad.

## Testing
It compiled.
As a borg, I was able to adjust suit sensors.
As any other kind of non-human mob, I was unable to adjust suit sensors.

## Changelog
:cl:
fix: Borgs are able to adjust suit sensors.
/:cl:

### CODE REVIEW

File: code/__DEFINES/mob_defines.dm [hunk 1]
```diff
@@ -249,6 +249,7 @@
 #define isgorilla(A) 		(istype((A), /mob/living/simple_animal/hostile/gorilla))
 #define ismorph(A)			(istype((A), /mob/living/simple_animal/hostile/morph))
 
+/// Checks if the target is any kind of silicon. isAI(), isrobot(), isdrone(), and ispAI() can be used for more specific checks.
```
Comment (DGamerL) [inline code/__DEFINES/mob_defines.dm:252]:
```suggestion
```
These are really not things that need commenting imo

File: code/modules/clothing/clothing.dm [hunk 2]
```diff
@@ -304,21 +313,19 @@
 	return TRUE
 
 /obj/item/clothing/under/proc/set_sensors(mob/user)
-	if(!user.Adjacent(src) || !ishuman(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
-		return
```
Comment (DGamerL) [inline code/modules/clothing/clothing.dm:308]: Will this allow you to open the TGUI menu at range?

File: code/modules/clothing/clothing.dm [hunk 3]
```diff
@@ -304,21 +313,19 @@
...
 	if(has_sensor <= SUIT_SENSOR_OFF)
-		to_chat(user, "This suit does not have any sensors.")
+		to_chat(user, "<span class='warning'>This suit does not have any sensors.</span>")
 		return
 
 	var/list/modes = list("Off", "Binary sensors", "Vitals tracker", "Tracking beacon")
 	var/switchMode = tgui_input_list(user, "Select a sensor mode:", "Suit Sensor Mode", modes, modes[sensor_mode + 1])
 	if(!user.Adjacent(src))
-		to_chat(user, "<span class='warning'>You have moved too far away!</span>")
+		to_chat(user, "<span class='warning'>You are too far away!</span>")
```
Comment (DGamerL) [inline code/modules/clothing/clothing.dm:326]: If the other comment was correct, please also revert this

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/modules/clothing/clothing.dm (hunk 2 above)
Comment (Fordoxia) [reply code/modules/clothing/clothing.dm:308]: No, I have tested. You cannot open the TGUI menu if you are not adjacent.

Comment (DGamerL) [reply code/modules/clothing/clothing.dm:308]: As said in discord, you can open the TGUI menu
image

File: code/modules/clothing/clothing.dm [hunk 4]
```diff
@@ -304,21 +313,24 @@
 	return TRUE
 
 /obj/item/clothing/under/proc/set_sensors(mob/user)
-	if(!user.Adjacent(src) || !ishuman(user) || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
+	if(!user.Adjacent(src))
+		to_chat(user, "<span class='warning'>You have moved too far away!</span>")
```
Comment (DGamerL) [inline code/modules/clothing/clothing.dm:317]: Message is wrong, also, add the other check back too

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/modules/clothing/clothing.dm (hunk 4 above)
Comment (Fordoxia) [reply code/modules/clothing/clothing.dm]: Done.

**Review by Sirryan2002 — APPROVED** [review]

**Review by Spaghetti-bit — APPROVED** [review]

**Comment by Fordoxia** [comment]
> This pull request seems to be stale as there have been no changes in 14 days, please make changes within 7 days or the PR will be closed. If you believe this is a mistake, please inform a development team member on Discord.

Leave this place.

**Comment by Sirryan2002** [comment]
@DGamerL

**Review by DGamerL — APPROVED** [review]
Needs a TM

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**